packages/calypso-e2e/src/lib/utils/get-test-account.ts

```diff
@@ -0,0 +1,70 @@
+type Env = 'edge' | 'production';
```

Comment:
Suggestion: `GutenbergEnvironment`.

If I understand correctly, this is to specify the target for Gutenberg build, correct?

Reply:
It could be used for Coblocks, too (see the last exported function).